src/components/DocSearchLogo.js

```diff
@@ -4,9 +4,10 @@ export function DocSearchLogo(props) {
   const { big = false } = props;
   return (
     <svg
-      width={big ? '50%' : '190'}
+      width={big ? (big === '100%' ? '100%' : '50%') : '190'}
```

Comment:
Instead of creating a `big` prop, we should spread the props to the SVG:

```js
function Component(props) {
  return (
    <svg {...props} />
  );
}
```

Reply:
Good point, used